test/upgrade/mzcompose.py

```diff
@@ -134,6 +144,7 @@ def test_upgrade_from_version(
         "testdrive-svc",
         "--no-reset",
         f"--var=upgrade-from-version={from_version}",
+        f"--var=upgrade-skip-if={str(check_from_accept_skip_ifs).lower()}",
```

Comment:
I am not sure if this will do what you intend to, but you can push the decision into the .td file so that you do not have to have any changes to the `mzcompose.py` file. Maybe some of those patterns will work:

```
$ skip-if
select split_part(ltrim(mz_version(), 'v'), '.', 1)::int = 0 and split_part(ltrim(mz_version(), 'v'), '.', 2)::int < 18;
```

or
```
$ skip-if
select split_part(ltrim(${from_version}, 'v'), '.', 1)::int = 0 and split_part(ltrim(${from_version}, 'v'), '.', 2)::int < 18;
```

Reply:
Strongly agree with pushing this logic into TD, but oof those version comparisons are brutal in SQL. We should think about introducing something like https://stackoverflow.com/a/52589801.

Reply:
hypothetically I need both version and this var, to distingiush the vX->current_source upgrade and the current_source->current_source upgrade